### DISCUSSION TITLE
Stereokit Browser

### DISCUSSION THREAD

**Discussion by notpatern** [StereoKit/StereoKit#1028] [Q&A]
Hi!
I'm new using your engine!
I wanted to know how to open a virtual keyboard when a fied is focused in an embedded browser.
I searched and didn't find anything.
Do you think you could help me?

Have a good day!

**Comment by maluoi** [comment]
Yeah, try [Platform.KeyboardShow](https://stereokit.net/Pages/StereoKit/Platform/KeyboardShow.html)! There's a small demo with that one over here: https://github.com/StereoKit/StereoKit/blob/master/Examples/StereoKitTest/Demos/DemoTextInput.cs

**Reply by notpatern** [reply]
> Yeah, try [Platform.KeyboardShow](https://stereokit.net/Pages/StereoKit/Platform/KeyboardShow.html)! There's a small demo with that one over here: https://github.com/StereoKit/StereoKit/blob/master/Examples/StereoKitTest/Demos/DemoTextInput.cs

thank you nick you're such a goat for this engine <3
